docs/modules/ROOT/pages/documentation/payara-server/admin-console/admin-console.adoc

```diff
@@ -7,3 +7,5 @@ Some of the enhancements in Payara Server relate to the Admin Console itself and
 xref:documentation/payara-server/admin-console/asadmin-recorder.adoc[Asadmin Recorder]:: Using the Admin Console to configure resources in Payara Server makes use of asadmin commands under the covers. Because of this fact, we are able to record all the asadmin commands that the Admin Console uses and output them to a file for later re-use.
 
 xref:documentation/payara-server/admin-console/environment-warning.adoc[Environment Warning]:: When working with Payara Server Community, it is very common to have multiple environments for various purposes like test or QA. It can often be all too easy to accidentally make changes to the wrong environment, so the Payara Server Web Administration console now allows you to add a configurable warning bar to give you an extra visual reminder of which environment you are working with.
+
+xref:documentation/payara-server/admin-console/auditing-service.adoc[Auditing Service]:: With the auditing active, you can collect information around the interactions with the Web Administration console and the _asadmin_ command-line tool usage. This can be used to keep track of changes and who made those changes to the Payara configuration. 
```

Comment:
```suggestion
xref:documentation/payara-server/admin-console/auditing-service.adoc[Auditing Service]:: With the auditing enabled, you can collect information around the interactions with the Web Administration console and the _asadmin_ command-line tool usage. This can be used to keep track of changes and who made those changes to the Payara Server configuration. 
```